src/components/ItemList.tsx

```diff
@@ -0,0 +1,26 @@
+import { News } from "../models/News";
```

Comment:
I'd rather have file names to be kebab (item-list). This is mainly due to issues with mac filesystem being case insensitive. I always avoid casing files in projects unless the project already has pascal-case set up

Reply:
I changed the file names, and added some new styles to the item component if you wanna check.